Light Vox Engine/Engine.cpp

```diff
@@ -249,24 +255,23 @@ LRESULT Engine::ProcessMessage( HWND hWindow, UINT message, WPARAM wParam, LPARA
     return Engine::engineInstance->HandleEvents( hWindow, message, wParam, lParam );
 }
 
-
 LRESULT Engine::HandleEvents( HWND hWindow, UINT message, WPARAM wParam, LPARAM lParam )
 {
     switch ( message )
     {
         //handle mouse down/up
         case WM_MOUSEMOVE:
-            OnMouseMove( wParam, GET_X_LPARAM( lParam ), GET_Y_LPARAM( lParam ) );
+            //OnMouseMove( wParam, GET_X_LPARAM( lParam ), GET_Y_LPARAM( lParam ) );
```

Comment:
nit: Can we just remove this code instead of commenting it out? I mean, source control exists.

If the comment is important, at least comment why it's commented